checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Remapper for Username Attribute
+      enabled: true
+      filter:
+        query: "@product:DLP"
+      processors:
+        - type: attribute-remapper
+          name: Map `user` to `usr.name`
+          enabled: true
+          sources:
+            - user
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `reject_category` to `checkpoint.quantum.firewall.reject_category`
+          enabled: true
+          sources:
+            - reject_category
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.reject_category
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: String Building for Source and Destination IP Port
+      enabled: true
+      filter:
+        query: '@product:("Firewall" OR "IPS" OR "Anti-Spam and Email Security" OR "URL
+          Filtering") '
+      processors:
+        - type: string-builder-processor
+          name: "Define `checkpoint.quantum.firewall.src_ip_port` equal to
+            %{network.client.ip}: %{network.client.port}"
+          enabled: true
+          template: "%{network.client.ip}:%{network.client.port}"
+          target: checkpoint.quantum.firewall.src_ip_port
+          replaceMissing: false
+        - type: string-builder-processor
+          name: "Define `checkpoint.quantum.firewall.dst_ip_port` equal to
+            %{network.destination.ip}: %{network.destination.port}"
+          enabled: true
+          template: "%{network.destination.ip}:%{network.destination.port}"
+          target: checkpoint.quantum.firewall.dst_ip_port
+          replaceMissing: false
+    - type: pipeline
+      name: Lookup for `app_risk` field
+      enabled: true
+      filter:
+        query: '@product:("Application Control" OR "URL Filtering") '
+      processors:
+        - name: Lookup for `app_risk` to `checkpoint.quantum.firewall.app_risk_str` field
+          enabled: true
+          source: app_risk
+          target: checkpoint.quantum.firewall.app_risk_str
+          lookupTable: |-
+            0, Unknown
+            1, Very low
+            2, Low
+            3, Medium
+            4, High
+            5, Critical
+            Unknown, Unknown
+            Very low, Very low
+            Low, Low
+            Medium, Medium
+            High, High
+            Critical, Critical
+          type: lookup-processor
+    - type: pipeline
+      name: Remapper for Status Attribute
+      enabled: true
+      filter:
+        query: '@product:"HTTPS Inspection"'
+      processors:
+        - type: attribute-remapper
+          name: Map `status` to `checkpoint.quantum.firewall.status`
```

Comment:
I asked it in a previous comment but could this attribute be converted to the standard status ?

Reply:
Reference: https://github.com/DataDog/integrations-core/pull/16522#discussion_r1521104726